README.md

```diff
@@ -251,12 +251,18 @@ await index.search(
     {
       "id": 2,
       "title": "Wonder Woman",
-      "genres": ["Action","Adventure"]
+      "genres": [
+        "Action",
+        "Adventure"
+      ]
```

Comment:
these are not specific linting rules of this repo, could you revert those changes?